OpenRA.Game/InstalledMods.cs

```diff
@@ -72,23 +72,27 @@ Manifest LoadMod(string id, string path)
 					throw new InvalidDataException(path + " is not a valid mod package");
 
 				package = new Folder(path);
-				if (!package.Contains("mod.yaml"))
-					throw new InvalidDataException(path + " is not a valid mod package");
```

Comment:
'common' mod does not have mod.yaml + 2 mods don't have 'icon.png' so on start this function throws 3 exceptions.
It is unnecessary, because this exception was ignored - catch returned null. Now the function has the same functionality without those exceptions.